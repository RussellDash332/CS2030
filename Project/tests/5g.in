2 0 2 1 5 9.0 1.0 1.0 0.0 0.0