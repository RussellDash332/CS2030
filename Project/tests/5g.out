0.000 1 arrives
0.000 1 serves by self-check 1
0.035 2 arrives
0.035 2 serves by self-check 2
0.046 3 arrives
0.046 3 waits at self-check 1
0.124 4 arrives
0.124 4 leaves
0.126 5 arrives
0.126 5 leaves
0.313 1 done serving by self-check 1
0.313 3 serves by self-check 1
2.680 2 done serving by self-check 2
3.015 3 done serving by self-check 1
[0.089 3 2]
